errors/dontpanic.go

```diff
@@ -0,0 +1,34 @@
+package errors
+
+var CodePanic Code = "PANIC"
+
+// DontPanic executes f and, if f panics, recovers from the panic
+// and returns the panic wrapped as an Error.
+func DontPanic(f func()) (err error) {
```

Comment:
This function name could be on the cover written with large, friendly letters =D

Reply:
I'm just happy that someone got this reference